packages/table/src/Table.tsx

```diff
@@ -14,6 +19,42 @@ const tableStyles = css`
   border-bottom: 1px solid ${uiColors.gray.light2};
 `;
 
+const ScrollState = {
+  None: 'none',
+  Left: 'left',
+  Right: 'right',
+  Both: 'both',
+} as const;
+
+type ScrollState = typeof ScrollState[keyof typeof ScrollState];
+
+function getScrollShadowStyle(scrollState: ScrollState): string {
+  const shadowColor = 'rgba(93,108,116,0.3)';
```

Comment:
I think this would be more legible as `transparentize(.7, uiColors.gray.dark1)`

Reply:
👍 but .7 looked too light so made it .6